### ISSUE TITLE
Washington Times

### ISSUE THREAD

**Issue by Wrench69** [reek/anti-adblock-killer#3895]
65.0.3325.51 beta (64-bits)
uBlock
Easylist + Anti-Adblock Killer | Reek + Malware domains
https://www.washingtontimes.com/
![wt](https://user-images.githubusercontent.com/31038127/36017249-37158808-0d77-11e8-9a46-a4e59e62d73d.jpg)

**Comment by uBlock-user** [comment]
Can't reproduce.

**Comment by SATGUY84** [comment]
Having the same problem with this site: https://www.washingtontimes.com/news/2018/feb/9/judge-upholds-termination-of-illegal-workers-disab/

**Comment by Ayelwen** [comment]
Having same issue. https://www.washingtontimes.com/news/2018/feb/19/columbine-survivor-bill-concealed-carry-schools/

I followed the instructions https://reek.github.io/anti-adblock-killer/

**Comment by mbierman** [comment]
Happens to me as well. Note at first the screen was blocked (and I could view the article) but shortly thereafter as I scrolled? or perhaps just based on time? Their block screen above reappeared. 

- Version: 64.0.3282.186 (Official Build) (64-bit)
- Adblock Plus: 1.13.5
- Tampermonkey: 4.5
-    Anti-Adblock Killer | Reek : '10.0

**Comment by uBlock-user** [comment]
Remove AAK and use uBlock Origin, it's patched there.

**Comment by mbierman** [comment]
Hmm thanks. But...
1. Are you saying it is the Ad blocker's fault or the userscript?
2. The thread above says that it wasn't reproducible but now it has been fixed?

**Comment by uBlock-user** [comment]
1) It's not patched in Easylist, even though it's been repeatedly reported, but uBO Filters did it a year ago. So why wait for something that will never happen ?

2) It's not reproducible on my end with uBO, because it's been patched in uBO Filters.

3) Regarding AAK - https://github.com/reek/anti-adblock-killer/wiki#notice

I won't say anything anymore. This is enough as an explanation to jump to uBO and abandon ABP.

**Comment by mbierman** [comment]
Thanks. I guess I am surprised that it is the ad blocker and not up to the userscript or filters but I take your point and that is all helpful to know. Thank you.

**Comment by mbierman** [comment]
Wait so are you saying that uBlock replaces the need for AAK (and a script manager) all together?

**Comment by uBlock-user** [comment]
Yes. For anti adblock purposes specifically and all other blocking needs. If you have uBlock, you don't need any third-party extensions or user-scripts like AAK.

**Comment by mbierman** [comment]
very cool! Thanks.

So is this now the place to report ublock bugs if I find any?

**Comment by uBlock-user** [comment]
AAK is already gone, I'm surprised that people still don't get it. It hasn't been updated in a year and yet there are so many users.

**Comment by uBlock-user** [comment]
For extension related bugs - https://github.com/gorhill/uBlock/issues

For anti adblock and all other blocking help/issues - https://github.com/uBlockOrigin/uAssets
